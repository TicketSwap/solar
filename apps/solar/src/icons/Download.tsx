import React from 'react'
import { SvgWrapper } from './SvgWrapper'

export const Download = (props: any) => (
  <SvgWrapper a11yTitle="Download" {...props}>
    <path d="M17 11h-2V4a1 1 0 1 1 2 0v7zm6 0c1.654 0 3 1.346 3 3v12c0 1.654-1.346 3-3 3H9c-1.654 0-3-1.346-3-3V14c0-1.654 1.346-3 3-3h6v8.836l-2.793-2.793a.997.997 0 0 0-1.414 0 .999.999 0 0 0 0 1.414l4.499 4.499a1.001 1.001 0 0 0 1.416 0l4.499-4.499a.999.999 0 1 0-1.414-1.414L17 19.836V11h6z" />
  </SvgWrapper>
)
