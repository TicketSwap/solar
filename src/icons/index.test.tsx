import React from 'react'
import { render } from '@testing-library/react'
import { Download } from './Download'

describe('Icon', () => {
  it('matches snapshot', () => {
    const { container } = render(<Download />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
