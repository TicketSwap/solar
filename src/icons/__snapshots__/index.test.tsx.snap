// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon matches snapshot 1`] = `
<span
  class="icon css-1l0cdq-Wrapper e1ouvt3m1"
>
  <svg
    aria-label="Download"
    class="css-1ciyvsb-InlineSvg e1ouvt3m0"
    clip-rule="evenodd"
    fill-rule="evenodd"
    focusable="false"
    preserveAspectRatio="xMidYMid meet"
    role="presentation"
    stroke-linejoin="round"
    stroke-miterlimit="1.414"
    viewBox="0 0 32 32"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M17 11h-2V4a1 1 0 1 1 2 0v7zm6 0c1.654 0 3 1.346 3 3v12c0 1.654-1.346 3-3 3H9c-1.654 0-3-1.346-3-3V14c0-1.654 1.346-3 3-3h6v8.836l-2.793-2.793a.997.997 0 0 0-1.414 0 .999.999 0 0 0 0 1.414l4.499 4.499a1.001 1.001 0 0 0 1.416 0l4.499-4.499a.999.999 0 1 0-1.414-1.414L17 19.836V11h6z"
    />
  </svg>
</span>
`;
